python-manual/modules/ROOT/pages/result.adoc

```diff
@@ -0,0 +1,236 @@
+= Manipulate query results
+
+In the xref:query-simple.adoc[basic query workflows] examples, transaction functions never returned a query result directly. Instead, they always _manipulated_ it in some way: either casting it to list, or calling a specific method. This is because the driver's output of a query is a `Result` object, which does not directly contain the result records. Rather, it encapsulates the Cypher result in a rich data structure that requires some parsing within the transaction function.
```

Comment:
```suggestion
In the xref:query-simple.adoc[basic query workflows] examples, transaction functions never returned a query result directly. 
Instead, they always _manipulated_ it in some way: either casting it to list or calling a specific method. 
This is because the driver's output of a query is a `Result` object, which does not directly contain the result records. 
Rather, it encapsulates the Cypher result in a rich data structure that requires some parsing within the transaction function.
```